examples/convecting_plankton.jl

```diff
@@ -45,15 +45,14 @@
 # We use a two-dimensional grid with 64² points and 1 m grid spacing:
 
 using Oceananigans
+using Oceananigans.Units
```

Comment:
Perhaps move this further down when first needed?

Reply:
Do we need a clarification about what does this import is used for?

Reply:
Maybe just making the names that are imported explicit:

```julia
using Oceananigans.Units: hour
```

Reply:
Yes I can explicitly import the names used.